### PR TITLE
Chore/liquidity provisiond stagnet1

### PR DESCRIPTION
**Pull request by malinantonsson**
# Related issues 🔗

Related to #1460 

# Description ℹ️
- Adds env file for stagnet1
- Updates env files to include stagnet1 in available networks

### CODE REVIEW

File: apps/liquidity-provision-dashboard/.env.mainnet [hunk 1]
```diff
@@ -2,7 +2,7 @@
 NX_VEGA_CONFIG_URL=https://static.vega.xyz/assets/mainnet-network.json
 NX_VEGA_URL=https://api.vega.xyz/query
 NX_VEGA_ENV=MAINNET
-NX_VEGA_NETWORKS='{\"MAINNET\":\"https://alpha.console.vega.xyz\"}'
+NX_VEGA_NETWORKS={\"MAINNET\":\"https://alpha.console.vega.xyz\",\"TESTNET\":\"https://console.fairground.wtf\",\"STAGNET1\":\"https://stagnet1.console.vega.xyz\",\"STAGNET3\":\"https://stagnet3.console.vega.xyz\"}
```
Comment (dexturr) [inline apps/liquidity-provision-dashboard/.env.mainnet:5]: chore (blocking): we do not want to point people to stagnet3 or stagnet1 on mainnet. These are internal test networks we do not wish to publicise.

**Review by dexturr — CHANGES_REQUESTED** [review]

File: apps/liquidity-provision-dashboard/.env.mainnet (hunk 1 above)
Comment (dexturr) [reply apps/liquidity-provision-dashboard/.env.mainnet:5]: Just fairground and mainnet please

File: apps/liquidity-provision-dashboard/.env.testnet [hunk 2]
```diff
@@ -2,7 +2,7 @@
 NX_VEGA_CONFIG_URL=https://static.vega.xyz/assets/testnet-network.json
 NX_VEGA_URL=https://api.n11.testnet.vega.xyz/graphql
 NX_VEGA_ENV=TESTNET
-NX_VEGA_NETWORKS='{\"MAINNET\":\"https://alpha.console.vega.xyz\"}'
+NX_VEGA_NETWORKS={\"TESTNET\":\"https://console.fairground.wtf\",\"STAGNET1\":\"https://stagnet1.console.vega.xyz\",\"STAGNET3\":\"https://stagnet3.console.vega.xyz\"}
```
Comment (dexturr) [inline apps/liquidity-provision-dashboard/.env.testnet:5]: same here

File: apps/liquidity-provision-dashboard/.env.mainnet (hunk 1 above)
Comment (malinantonsson) [reply apps/liquidity-provision-dashboard/.env.mainnet]: That makes a lot of sense now that you say it 🤦‍♀️ thank you!

**Pull request merged**